package.json

```diff
@@ -21,58 +21,66 @@
   "engines": {
     "node": "14"
   },
-  "devDependencies": {
+  "dependencies": {
```

Comment:
All of the dependencies were in the `"devDependencies"` object, and there was no `"dependencies"` object.

So create a `"dependencies"` object and moved the production dependencies there.

Reply:
There's no intended change, other than moving the dependencies from one object to the other.